packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -16,8 +20,23 @@ const AvatarContainer = styled.div`
   float: left;
 `;
 
-const OpenQueueCard = ({ queue }: OpenQueueCard) => {
+const EditOutlinedFloatedRight = styled(EditOutlined)`
+  float: right;
+`;
+
+const OpenQueueCard = ({ queue, isTA }: OpenQueueCard) => {
+  const [editingNotes, setEditingNotes] = useState(false);
   const staffList = queue.staffList;
+
+  const updateNotes = (value: any) => {
+    const notes = (value.target as any).value;
```

Comment:
we probably wanna make it a controlled component because right now if you put in a note, and then edit it, it will show an empty input.

Reply:
Good call. I just changed the input to use `updatedNotes` state instead